openmetadata-docs/content/v1.5.x-SNAPSHOT/connectors/database/greenplum/index.md

```diff
@@ -137,6 +138,24 @@ In order to integrate SSL in the Metadata Ingestion Config, the user will have t
 
 {% /stepsContainer %}
 
+## Securing Greenplum Connection with SSL in OpenMetadata
+
+To establish secure connections between OpenMetadata and a Greenplum database, you can configure SSL using different SSL modes provided by Greenplum, each offering varying levels of security.
+
+Under `Advanced Config`, specify the SSL mode appropriate for your connection, such as `prefer`, `verify-ca`, `allow`, and others. After selecting the SSL mode, provide the CA certificate used for SSL validation (`caCertificate`). Note that Greenplum requires only the CA certificate for SSL validation.
+
+{% note %}
+
+For IAM authentication, it is recommended to choose the `allow` mode or another SSL mode that fits your specific requirements.
+
+{% /note %}
+
+{% image
+  src="/images/v1.4/connectors/ssl_connection.png"
```

Comment:
here and other places as well  for `v1.5`.

Reply:
@SumanMaharana Update it to v.1.5.